frontend/src/metabase/nav/components/ProfileLink/ProfileLink.jsx

```diff
@@ -60,14 +75,10 @@ function ProfileLink({ user, adminItems, onLogout }) {
         link: "/admin",
         event: `Navbar;Profile Dropdown;Enter Admin`,
       },
-      {
+      helpLink.visible && {
```

Comment:
I'm on the fence about this. I feel like we should just inline `helpLinkSetting !== "hidden"` here. And `getHelpLink` or `useGetHelpLink` could just return the link instead.

Maybe that's more readable because we don't need to understand that function (or hook) return an object which has 2 keys `visible` and `href`.  What do you think?